x-pack/plugins/console_extensions/spec/overrides/xpack.ml.get_overall_buckets.json

```diff
@@ -0,0 +1,13 @@
+{
+  "xpack.ml.get_overall_buckets": {
+    "data_autocomplete_rules": {
+      "allow_no_jobs": [true, false],
```

Comment:
Is the pattern `[true, false]` used to define values to suggest within the array?